src/Stratis.Bitcoin.IntegrationTests/Extensions/CoreNodeExtensions.cs

```diff
@@ -12,13 +13,16 @@ public static class CoreNodeExtensions
     {
         public static Money GetProofOfWorkRewardForMinedBlocks(this CoreNode node, int numberOfBlocks)
         {
-            var powValidator = node.FullNode.NodeService<IPowConsensusValidator>();
-            var halvingInterval = powValidator.ConsensusParams.SubsidyHalvingInterval;
+            var coinviewRule = node.FullNode.NodeService<IRuleRegistration>().GetRules().OfType<PowCoinviewRule>().Single();
```

Comment:
Unvar please

Reply:
type is on the right - therefore complies with policy  (.... I'll duck for cover)

Reply:
only if type is after `new` or `as Type` or `(Type)x`

`var x = y<Type1>.Smth<Type2>().DoMagic<Type3>().Single()` doesn't count.

Reply:
This is not about Var or not here. A puppy died when this line was written.

Reply:
`var` is fine here

Reply:
>This is not about Var or not here. A puppy died when this line was written.
we have a little wrapper for it , GetRule.
However, probably in this context in the test it can probably just new up the rule directly

Reply:
😄 it was an evil puppy